src/content/en/updates/2019/02/intersectionobserver-v2.md

```diff
@@ -0,0 +1,210 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Augment IntersectionObserver to report information about occlusion and visual effects.
+
+{# wf_updated_on: 2019-02-11 #}
+{# wf_published_on: 2019-02-06 #}
+{# wf_tags: intersectionobserver #}
+{# wf_featured_image: /web/updates/images/generic/timeline.png #}
+{# wf_featured_snippet: Lorem ipsum. #}
+{# wf_blink_components: Blink>Layout #}
+
+# Trust is Good, Observation is Better—Intersection Observer&nbsp;v2 {: .page-title}
+
+{% include "web/_shared/contributors/thomassteiner.html" %}
+
+<div class="clearfix"></div>
+
+Intersection Observer&nbsp;v1 is one of these APIs that's probably universally loved, and, now that
```

Comment:
"one of these APIs" reads a little oddly for American English. I'd suggest changing it to "one of those APIs".